src/urllib3/connection.py

```diff
@@ -130,6 +130,8 @@ class HTTPConnection(_HTTPConnection):
     _tunnel_port: int | None
     _tunnel_scheme: str | None
 
+    last_activity: datetime.datetime
```

Comment:
Don't you also need idle_timeout here too?

Reply:
the linter didn't complain about that one but yes, I added it now